src/components/DateInput/styles.ts

```diff
@@ -3,31 +3,35 @@ import COLORS from '@/styles/colors';
 import { H4 } from '@/styles/text';
 import { openSans } from '@/styles/fonts';
 
-const getBorderColor = (error: string, filled: boolean): string => {
-  if (error !== '') return COLORS.redMid;
-  return filled ? COLORS.greyMid : COLORS.greyLight;
-};
-
 export const InputLabel = styled(H4)`
   color: ${COLORS.greyDark};
-  margin-bottom: 0.625rem; //10px
+  margin-bottom: 10px;
 `;
 
-export const InputDate = styled.input<{ $error: string; $filled: boolean }>`
+export const InputDate = styled.input<{ $error: boolean; $filled: boolean }>`
   ${openSans.style}
-  font-size: 0.875rem; //14px
+  font-size: 0.875rem; // 14px
   font-weight: 400;
   cursor: text;
-  color: ${({ $filled }) =>
-    $filled ? COLORS.greyDarker : COLORS.placeholderText};
-  border-radius: 0.313rem; // 5px
-  border: 2px solid ${({ $error, $filled }) => getBorderColor($error, $filled)};
+  border-radius: 5px;
```

Comment:
generally `px` for most things. I didn't want to make that change in the `TextInput` as well as I didn't need to touch its styles, but since I touched the markup, I might as well.

I would be opposed to making that change across all styles for now though, this PR is big enough as is....